cmd/shcd.go

```diff
@@ -85,25 +85,25 @@ var shcdCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		if v.IsSet("hmn-connections") {
-			err := createHMNSeed(shcd.Topology)
+		if createHMN {
```

Comment:
The variables are always "set" since they have a default, so checking the actual bool value of the variable fixes the check.